serverless/deploy/docker/witness/docker-compose.yaml

```diff
@@ -21,8 +21,8 @@ services:
       - "--config_file=/witness-config/${WITNESS_CONFIG_FILE:-witness.config}"
       - "--logtostderr"
     restart: always
-    ports:
-      - "8100:8100"
+    expose:
```

Comment:
Doesn't this have the downside that if you want to bring up a witness and it actually work as its own distributor then its API isn't exposed? I feel like that's a more common default use-case than running multiple containers. Let's chat about this tomorrow, there must be a reasonable compromise here.